src/index.tsx

```diff
@@ -1,15 +1,40 @@
 import './wdyr'
 import 'lib/polyfills'
 
+import * as Sentry from '@sentry/react'
 import { App } from 'App'
 import { AppProviders } from 'AppProviders'
+import { getConfig } from 'config'
 import React from 'react'
 import { createRoot } from 'react-dom/client'
 import { renderConsoleArt } from 'lib/consoleArt'
 import { reportWebVitals } from 'lib/reportWebVitals'
 
 import * as serviceWorkerRegistration from './serviceWorkerRegistration'
 
+Sentry.init({
+  ...(window.location.hostname === 'localhost'
+    ? {}
+    : { dsn: getConfig().REACT_APP_SENTRY_DSN_URL }),
+  attachStacktrace: true,
+  denyUrls: ['alchemy.com'],
+  integrations: [
+    // Sentry.browserTracingIntegration(),
+    // Sentry.replayIntegration(),
+    Sentry.httpClientIntegration({
+      failedRequestStatusCodes: [[400, 599]],
+    }),
+  ],
+  beforeSend(event) {
+    // https://github.com/getsentry/sentry-javascript/issues/8353 / https://forum.sentry.io/t/turn-off-event-grouping/10916/3
+    event.fingerprint = [(Math.random() * 1000000).toString()]
+    return event
+  },
+  enableTracing: true,
+  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+  tracePropagationTargets: ['localhost'],
```

Comment:
```
     * By default the SDK will attach those headers to all requests to localhost
     * and same origin.
```

It sounds like `localhost` is included by default anyway, but not harm in being explicit here.